src/gamma.jl

```diff
@@ -2,7 +2,7 @@
 
 using Base.MPFR: ROUNDING_MODE, big_ln2
 
-@compat ComplexOrReal{T} = Union{T,Complex{T}}
+ComplexOrReal{T} = Union{T,Complex{T}}
```

Comment:
even though not necessary maybe add a `const` to make it evident it's a type alias?